core/setup.py

```diff
@@ -73,6 +73,8 @@ def read(fname):
         'networkx>=2.3,<3',
         'packaging~=20.9',
         'sqlparse>=0.2.3,<0.4',
+        'tree-sitter==0.19.0',
+        'tree-sitter-jinja2==0.1.0a1',
```

Comment:
I know I mentioned before that this should probably be an "opt-in" dependency, but I think I'm leaning towards not doing that. I'm pretty happy with how we packaged this up, so I'm down to keep this in the list of required dependencies! 👍

Reply:
That's good by me! If we feel good about this installing on all supported systems/architectures/python versions, let's include it. We can call it out in the RC and see if we hear back about any installation issues.

Reply:
I want to avoid making this a special case if we can. Plus if people have install issues with the RC (which I don't expect) we can fix them before the general release.